### DISCUSSION TITLE
PCL II 2.6.1？

### DISCUSSION THREAD

**Discussion by MoYuan-CN** [Meloong-Git/PCL#1898] [提问] [answered]
https://github.com/Hex-Dragon/PCL2/issues/1892#issuecomment-1563204746
如上帖，看到了我就去尝试更新下我的快照启动器，但是。。。
![image](https://github.com/Hex-Dragon/PCL2/assets/109946426/ee3fc860-4955-499b-92b6-c9ec0c723754)
啥情况嘞

**Comment by WForst-Breeze** [comment, marked as answer]
2.6.1 仅限正式版，快照版无需更新
![image](https://github.com/Hex-Dragon/PCL2/assets/110760354/a9b87ae5-f610-45c0-8561-7275540c4d2d)

**Reply by MoYuan-CN** [reply]
哦哦
